public/scripts/SearchResults/result-view-controller.js

```diff
@@ -29,8 +29,8 @@ export default class ResultViewController {
 
 
         let dlIcon = publisherDetails.nextElementSibling.firstElementChild;
-        const urls = [record.urlPDF, record.urlIIIF, record.urlOther];
-        for(let i = 0; i < 3; i++) {
+        const urls = [record.urlPDF, record.urlIIIF, record.urlPlainText, record.ALTOXML, record.urlOther];
```

Comment:
```suggestion
        const urls = [record.urlPDF, record.urlIIIF, record.urlPlainText, record.urlALTOXML, record.urlOther];
```

This just looks like a typo - have you been able to test an ALTO XML record?

Reply:
Well caught! No, we don't have any yet to test against. I also spotted I left the alt tags empty, so will correct that too.